ouroboros-network/src/Ouroboros/Network/NodeToClient/Version.hs

```diff
@@ -30,7 +30,9 @@ data NodeToClientVersion
     | NodeToClientV_2
     -- ^ added local-query mini-protocol
     | NodeToClientV_3
-    -- ^ enabled @CardanoNodeToClientVersion3@
+    -- ^ enabled @CardanoNodeToClientVersion2@
+    | NodeToClientV_4
```

Comment:
@coot FYI, this means we're going to start using `NodeToClientV_4` in the next release.